config/locales/pl.yml

```
@@ -17,19 +17,20 @@ pl:
         instructor: Instruktor
+        created_at: Stworzono
+        description: Opis
         course_content: Kontent
         edit_content: Edytuj kontent
         no_content_yet: Nie masz jeszcze żadnego kontentu.
         course_settings: Ustawienia
         edit_settings: Edytuj ustawienia
         no_settings_configured: Brak ustawien
-        description: Opis
-        no_description_yet: Nie masz jeszcze opisu.
+        course_details: Informacje ogólne
```

YAMLlint (1.35.1)

[error] duplication of key "course_details" in mapping

(key-duplicates)